src/DurableTask.ServiceBusAMS/ServiceBusOrchestrationService.cs

```diff
@@ -530,12 +528,19 @@ public int GetDelayInSecondsAfterOnFetchException(Exception exception)
         /// <param name="cancellationToken">The cancellation token to cancel execution of the task</param>
         public async Task<TaskOrchestrationWorkItem> LockNextTaskOrchestrationWorkItemAsync(TimeSpan receiveTimeout, CancellationToken cancellationToken)
         {
-            ServiceBusSessionReceiver receiver = await serviceBusClient.AcceptNextSessionAsync(this.orchestratorEntityName);
-            if (receiver == null)
+            ServiceBusSessionReceiver receiver = null;
+            try
             {
-                return null;
+                receiver = await serviceBusClient.AcceptNextSessionAsync(this.orchestratorEntityName);
+            } catch (ServiceBusException ex)
```

Comment:
please add some logging here, why are we swallowing the exception and not throwing here?

Reply:
like if there was any sb exception receiver would be null and we would get a null ref below. Without any logging this will be hard to debug

Reply:
Will add to the next PR